### ISSUE TITLE
Named argument is recorded as local symbol of wrong scope

### ISSUE THREAD

**Issue by mlangkabel** [CoatiSoftware/SourcetrailPythonIndexer#31]
Consider the following code snippet:
```python
def foo(bar):
	pass

def baz():
	foo(bar = True)
	bar = True
```
Here the indexer merges the occurrences of the two tokens calles `bar` inside the function definition of `baz`, even though the first of these tokens should actually be an occurrence of the parameter `bar` of the function `foo`.

**Comment by mlangkabel** [comment]
Important note for fixing this issue: these named arguments may also be unsolved.

**Comment by mlangkabel** [comment]
Fixed with merge of pull request #35.
